altimeter_ms5607_spi/altimeter_ms5607_spi.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <math.h>
+
+#if defined(STM32F1)
+#include "stm32f1xx_hal.h"
+#endif
+
+#ifndef SERIAL_TIMEOUT
+#define SERIAL_TIMEOUT 10
+#endif
+
+class AltimeterMs5607Spi {
+  public:
+    struct Data {
+        // temperature data in degrees celsius
+        double temperature = std::nan("");
```

Comment:
Why do these need to be NaN by default?

Reply:
NaN indicates no valid data, not (?) technically required in this case?